activerecord/lib/active_record/nested_attributes.rb

```diff
@@ -236,6 +236,24 @@ class TooManyRecords < ActiveRecordError
     # of hashes can be used with hashes generated from HTTP/HTML parameters,
     # where there may be no natural way to submit an array of hashes.
     #
+    # === Updating a Collection and Its Attributes
```

Comment:
Should this be `It's Attributes`? 🤔

Reply:
It's is short for "it is". Its is the possessive form. In this case, we're describing the collection's attributes.